ecl/provider_connectivity/v2/_proxy.py

```diff
@@ -152,6 +152,15 @@ def operations(self, **params):
         return list(self._list(_operation.Operation,
                                paginated=False,
                                **params))
+    def get_operation(self, operation_id):
+        """Show operation.
+
+        :param operation_id: operation unique id
+        :returns: A show of operation objects
+        :rtype: show of :class:`~ecl.provider_connectivity.v2.operation.
+            Operation`
+        """
+        return self._get(_operation.Operation,operation_id)
```

Comment:
```suggestion
        return self._get(_operation.Operation, operation_id)
```
引数と引数との間は半角スペースを入れてください。

Reply:
おっしゃる通り修正しました。